models/reviews.models.js

```diff
@@ -35,3 +35,21 @@ exports.updateVotes = (reviewId, incVotes) => {
       }
     });
 };
+
+// ALMOST WORKS BUT JUST MISSING THE comment_count KEY.
+// exports.fetchReviews = () => {
+//   return db.query(`SELECT * FROM reviews`).then((reviews) => {
+//     console.log(reviews.rows, "<--- reviews.rows");
+//     return reviews.rows;
+//   });
+// };
+
+exports.fetchReviews = (sortBy = "date", order = "DESC") => {
+  let queryStr = `SELECT reviews.owner, reviews.title, reviews.review_id, reviews.category, reviews.review_img_url, reviews.created_at, reviews.votes, 
```

Comment:
You could make this dry-er by using reviews.* instead of writing each column name

Reply:
Can I merge after fixing this?